software/jupyter-notebook.md

```diff
@@ -25,8 +25,17 @@ Please see our guide on using conda and [how to transfer files]({{site.baseurl}}
 Usually, for most of your work you should store your files at `/fs1/projects/{project-name}/`, which lives on the parallel file-system storage. You can also use your home directory (`/home/{username}/`) for quick experiments and convenient  access to scripts, but keep in mind that your home directory has limited storage space and performance. The parallel file-system storable is much faster and has way more space for your notebooks and data.
 ```
 
+### SSH Port Forwarding
+
+As the compute nodes where workloads run on the cluster are not directly reachable from the campus network, you'll need to use SSH port forwarding through the login node to access your Jupyter Notebook instances on the cluster. Also, as the login node itself is not currently reachable off campus, either SSH port forwarding through the Linux lab machines or VPN access is needed to access the login node when off campus.
```

Comment:
where workloads run on the cluster should be in parenthesis:

As the compute nodes (where workloads run on the cluster) are not directly reachable from the campus network,

Reply:
Thank you. Resolved (cee89976c60caabf55fee522e3fca0259aa37f4e)